src/uu/comm/src/comm.rs

```diff
@@ -76,6 +150,26 @@ fn comm(a: &mut LineReader, b: &mut LineReader, delim: &str, opts: &ArgMatches)
     let mut total_col_2 = 0;
     let mut total_col_3 = 0;
 
+    let check_order = opts.get_flag(options::CHECK_ORDER);
+    let no_check_order = opts.get_flag(options::NO_CHECK_ORDER);
+
+    // Skip order checking if files are identical and --check-order is not specified
+    let skip_order_check = no_check_order
```

Comment:
This variable is always used in the negated form `!skip_order_check`. To avoid the double negation I would rename the variable and make it about whether to run an order check instead of whether to skip an order check.